src/Sylius/Behat/Context/Api/Shop/RegistrationContext.php

```diff
@@ -186,4 +225,28 @@ private function assertFieldValidationMessage(string $path, string $message): vo
             $decodedResponse['violations']
         );
     }
+
+    private function fillContent(?string $email = 'example@example.com', ?string $password = 'example'): void
+    {
+        $this->content = [
+            'firstName' => 'First',
+            'lastName' => 'Last',
+            'email' => $email,
+            'password' => $password
+            ];
+    }
+
+    private function enableAccount(string $email): void
```

Comment:
I'm not convinced to this method, this is a logic that should be implemented in backend, not just in test behaviour. In the UI context, there is a listener (`UserRegistrationListener`) that automatically enables an account if account verification on a current channel is not required or sends an email. As long as we don't handle emails in API, I would be for enabling an account depending on channel in some listener or even in registration handler for now.